website/docs/faqs/Git/git-migration.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "How to migrate git providers"
+sidebar_label: "Git Migration"
+id: "git-migration"
+description: "Learn how to migrate git providers in dbt Cloud with minimal disruption."
+tags: [Git]
+---
+
+To migrate from one git provider to another, refer to the following steps to avoid minimal disruption:
+
+1. Outside of dbt Cloud, you'll need to import your existing repo into your new provider. 
```

Comment:
Do we use "repo" in other parts of the docs or do we usuall refer to "repository". (repo appears a few times here)

Reply:
we do use repo all over the docs, you can shorten words if you introduce the main word at the beginning. but I'll update to repository for consistency on this page.